calling/src/androidMain/kotlin/com/wire/kalium/calling/Calling.kt

```diff
@@ -127,6 +128,12 @@ interface Calling : Library {
         clientsJson: String
     )
 
+    @Suppress("FunctionNaming", "FunctionParameterNaming")
+    fun wcall_set_active_speaker_handler(
+        inst: Handle,
+        wcall_active_speaker_h: ActiveSpeakersHandler
```

Comment:
Could you a different name  instead of `wcall_active_speaker_h`, `activeSpeakersHandler`  ?

Reply:
As this is where we are mapping the native functions I don't think we can rename them 🤔